lib/helpers.js

```diff
@@ -363,7 +363,8 @@ exports.getEventPermissions = (data) => {
     }
 
     permissions.manage_question_lines = hasPermission(corePermissions, 'global:manage_question_lines:' + event.type);
-    permissions.see_questions = permissions.manage_question_lines || (myApplication ? myApplication.confirmed : false);
+    permissions.see_question_lines = hasPermission(corePermissions, 'global:see_question_lines:' + event.type);
```

Comment:
do you use it in code later?

Reply:
What do you mean? If I would use this see_question_lines permission somewhere else as well?

Reply:
I mean, for `see_questions`, there's this check in `listAllQuestionLines`:

```
exports.listAllQuestionLines = async (req, res) => {
    if (!req.permissions.see_questions && !req.permissions.manage_question_lines) {
        return errors.makeForbiddenError(res, 'You are not allowed to see questions');
    }

```

I cannot see the check like this or any usage of `see_question_lines`, that's why I don't know if it's used somewhere (probably not), and if not, you should add its usage somewhere

Reply:
Ah, that's a good one. I will check again later and see what can be done

Reply:
I looked at this, and since see_question_lines is only used in see_questions this is all fine.